contents/swift/welcome-to-swift.md

```diff
@@ -313,7 +358,26 @@ Reference cycles are bad, because they cause memory leaks. Even though A and B a
 
 Here is an [article](https://krakendev.io/blog/weak-and-unowned-references-in-swift) with greater in-depth explanation and examples.
 
-## CocoaPods
+### Benefit: Reuse Existing Code Easily
+
+[CocoaPods](https://guides.cocoapods.org/using/getting-started.html) is a dependency manager for Swift and Objective-C Cocoa projects which has over 58 thousand libraries and is used in over 3 million apps. Instead of reinventing the wheel, you can check this out to obtain code that helps resolve common issues. If you have done something new with Swift, you can also make your code into a library with CocoaPods for others to use!
+
+## How to Get Started
+
+A Macbook is required for Swift development, but an iPhone or iPad are not. The Swift IDE `X-Code` has built-in simulators for all mobile devices.
+
+If you have not learnt any other programming languages before, this [Game App](https://www.apple.com/swift/playgrounds/) could be a good choice to learn swift as well as programming.
+
+If you have been familiar with some programming language, 
+reading the [Language Guide in the official documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/) is recommended, since it explains everything quite clearly, albeit quite verbose.
+
+If you are really pressed for time, here are a couple of cheatsheets with code examples:
+
+- [Swift Cheat Sheet](https://mhm5000.gitbooks.io/swift-cheat-sheet/content/index.html) (web-friendly)
+- [Swift 3.0 Cheat Sheet and Quick Reference](https://koenig-media.raywenderlich.com/uploads/2014/06/RW-Swift-Cheatsheet-0_7.pdf) (print-friendly)
+
+## Where to Go from Here
```

Comment:
Can remove this section. The article is meant for those who don't know Swift, which means `Getting started` is where they need to go. There is no need for this section.